eng/common/TestResources/New-TestResources.ps1

```diff
@@ -156,7 +156,7 @@ if ($ProvisionerApplicationId) {
     }
 
     $provisionerAccount = Retry {
-        Connect-AzAccount -Tenant $TenantId -Credential $provisionerCredential -ServicePrincipal -Environment $Environment @subscriptionArgs
+        Connect-AzAccount -Force -Tenant $TenantId -Credential $provisionerCredential -ServicePrincipal -Environment $Environment @subscriptionArgs
```

Comment:
Nit: generally more common to put -Force last, though it doesn't change anything other than put more meaningful parameters up front.

Reply:
Can you change this in the other PR you just merged? (Rather, the files that were there.)

Reply:
Tracked here: https://github.com/Azure/azure-sdk-tools/issues/622